lstchain/scripts/lstchain_r0_to_r0g.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+"""
+Script to apply gain selection to the raw fits.fz produced by EvB6 in CTAR1
+format (aka R1v1).
+
+Gain selection is only applied to shower events (not to interleaved pedestal
+and flatfield)
+
+It uses heuristic identification of the interleaved flatfield (FF) events (
+given the occasional problems we have with the FF event tagging)
+
+"""
+import logging
+import protozfits
+import argparse
+
+from ctapipe.io import EventSource
+from ctapipe.containers import EventType
+from lstchain.io import standard_config
+from traitlets.config import Config
+
+import numpy as np
+from contextlib import ExitStack
+
+parser = argparse.ArgumentParser(description="Gain selector program (R0 to "
+                                             "R0G)")
+parser.add_argument('-f', '--R0-file', dest='input_file',
+                    type=str, help='Input R0 file name (of stream 1)')
+
+parser.add_argument('-o', '--output-dir', dest='output_dir',
+                    type=str, default='./',
+                    help='Output directory')
+
+# Range of waveform to be checked (for gain selection & heuristic FF
+# identification)
+SAMPLE_START = 3
+SAMPLE_END = 39
+
+def main():
+    args = parser.parse_args()
```

Comment:
I think we should add a logging config that at least logs the timestamps by default.

Reply:
Can you clarify the thing about the timestamps?

Reply:
that the logging format should contain the timestamp of each message. Default format is: `BASIC_FORMAT = "%(levelname)s:%(name)s:%(message)s"`, then it should include also `%(asctime)s` (timestamp)